source/blog/2018-07-20-the-ember-times-issue-56.md

```diff
@@ -1,93 +0,0 @@
----
-title: The Ember Times - Issue No. 56
```

Comment:
I don't think we should delete this file or it will remove the last Issue # 56 from the blog page?

Reply:
Yes you're right, this file should still be kept :+1:

Reply:
I added the issue back in with the PR #3477